tests/CiIntegrationTest.php

```diff
@@ -168,6 +163,80 @@ public function testIntegration()
         $this->assertSame(0, $result3['code']);
     }
 
+    public function testIntegrationWithChangesToConfigFile()
+    {
+        static::executeCommand(implode(' && ', array(
+            'git checkout -b changes-to-config-file -q',
+            'echo "" >> dir\ b/file\ b.php',
+            'echo "echo 1;" >> dir\ b/file\ b.php',
+            // `sed -i ...` is not handled the same on Linux and macOS
+            'sed -e \'s/@Symfony/@PSR2/\' .php_cs.dist > .php_cs.dist.new',
+            'mv .php_cs.dist.new .php_cs.dist',
+            'git add .',
+            'git commit -m "Random changes including config file" -q',
+        )));
+
+        $integrationScript = explode("\n", str_replace('vendor/bin/', './../../../', file_get_contents(__DIR__.'/../dev-tools/ci-integration.sh')));
+        $steps = array(
```

Comment:
logic of this test is same as prev one.
you just need data provider for branch-case and expected outputs